src/pages/MyGoalsPage/MyGoalsPage.tsx

```diff
@@ -39,7 +39,7 @@ export const MyGoalsPage = () => {
 
   const [loading, setLoading] = useState(false);
   const [activeGoals, setActiveGoals] = useState<GoalItem[]>([]);
-  const [archivedGoals, setArchivedGoals] = useState<GoalItem[]>([]);
+  const [doneGoals, setDoneGoals] = useState<GoalItem[]>([]);
```

Comment:
Please revert the naming convention to archived. As in the schema we are following the same name. Renaming "archive" => "done" was only for UI not for variables and states in our code.

Reply:
Reverted the commit as you mentioned